client/sites-dashboard/hooks/use-upsell-info.ts

```diff
@@ -0,0 +1,7 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+
+export function useUpsellInfo( site: SiteExcerptData ): string {
+	const { __ } = useI18n();
+	return site.is_wpcom_atomic ? __( 'Included in your plan' ) : __( 'Requires a Business Plan' );
```

Comment:
I think 'Included in your plan' is a little bit much:

<img width="564" alt="image" src="https://user-images.githubusercontent.com/36432/221614905-adb3bf62-78ba-4d17-b1bb-623ee86e4ce1.png">

AFAIK, we don't have that pattern elsewhere in the UI.

Also, a Simple site may have a Business plan so `site.is_wpcom_atomic` isn't necessarily the most appropriate check here.

What about something like this? Maybe with an icon?

<img width="550" alt="image" src="https://user-images.githubusercontent.com/36432/221618801-2b9f875f-07f9-4aaf-82bf-804e05819d37.png">

Reply:
> Also, a Simple site may have a Business plan so site.is_wpcom_atomic isn't necessarily the most appropriate check here.
Yes!, I need to improve it. I'll check if the site has the SFT feature available.

> What about something like this? Maybe with an icon?

Yes!, I envision it to be something similar to the jetpack icon.

Reply:
What do you think about doing something like this instead? The button would direct the customer to the Hosting Configuration page that explains what are the features included.

![image](https://user-images.githubusercontent.com/4933/221642137-99450857-17d0-4a89-9471-9b67da792031.png)

Reply:
I want to suggest another alternative. We display the star icon for free plans and the  `Requires Business Plan` text when the mouse hovers over the icon to explain what it means.
We use the `info` menu item text to increase awareness of the dev features. `SSH, DDBB, GitHub …`

@javierarce , what do you think?

- I've created this PR to play with this solution https://github.com/Automattic/wp-calypso/pull/73934

## Screencast:

https://user-images.githubusercontent.com/779993/222123967-38aec15c-957b-4f7a-aabc-c6639bf4730f.mp4



## Screenshots

| **Free** | **Business** |
|---|---|
| <img width="370" alt="Screenshot 2023-03-01 at 11 15 13" src="https://user-images.githubusercontent.com/779993/222124803-743b232d-cc7d-454f-b6ca-45fec81ad9b4.gif"> | <img width="370" alt="Screenshot 2023-03-01 at 11 15 13" src="https://user-images.githubusercontent.com/779993/222124418-ee578453-e8f6-4ced-9300-f04a104d0e06.png"> |

Reply:
I'm afraid I don't like this solution. Even if we tweaked the design, the use of the icon is very redundant and requires you hover/select the item to read the upsell message.

Reply:
> ![image](https://user-images.githubusercontent.com/4933/221665936-0121104b-36a4-436e-917b-4a45a0cb9c56.png)

My only concern about this approach is that we are not applying the same pattern for `Plugins`, which requires a Business plan too.

Reply:
A less repetitive option.

@javierarce, let's talk in sync to find the best solution 💪 

| **Free** | **Business** |
|---|---|
| <img width="395" alt="Screenshot 2023-03-01 at 14 22 26" src="https://user-images.githubusercontent.com/779993/222169782-8b406908-2e3d-4805-8eac-5b4bd2a3b600.png"> | <img widht="195" src="https://user-images.githubusercontent.com/779993/222124418-ee578453-e8f6-4ced-9300-f04a104d0e06.png" /> |